crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -387,10 +387,19 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                     }
                 };
 
+                let signing_key_bytes: [u8; 32] =
+                    signing_key_bytes.as_slice()[0..32].try_into().unwrap();
+                let x_coordinate_bytes: [u8; 32] = verifying_key_bytes.iter().as_slice()[1..33]
+                    .try_into()
+                    .unwrap();
+                let y_coordinate_bytes: [u8; 32] = verifying_key_bytes.iter().as_slice()[33..65]
+                    .try_into()
+                    .unwrap();
+
```

Comment:
Let's adjust the let above so it already returns the correct types instead of addign this conversion step.

Reply:
Okay, I get what you mean.

Reply:
Updated! @cptartur